README.md

```diff
@@ -145,13 +144,13 @@ You can now pretokenize your data using `tools/preprocess_data.py`.
 Usage:
 
 ```
-preprocess_data.py [-h] --input INPUT [--json-keys JSON_KEYS [JSON_KEYS ...]] [--split-sentences] [--keep-newlines] --tokenizer-type {BertWordPieceLowerCase,BertWordPieceCase,GPT2BPETokenizer} [--vocab-file VOCAB_FILE] [--merge-file MERGE_FILE] [--append-eod]
+preprocess_data.py [-h] --input INPUT [--jsonl-keys JSONL_KEYS [JSONL_KEYS ...]] [--split-sentences] [--keep-newlines] --tokenizer-type {BertWordPieceLowerCase,BertWordPieceCase,GPT2BPETokenizer} [--vocab-file VOCAB_FILE] [--merge-file MERGE_FILE] [--append-eod]
```

Comment:
From context, this section is supposably program output from <samp>preprocess_data.py</samp>, but it is not here to review. The script should either be modified so program output coincides with what is now documented here, or these changes should be reverted.

Reply:
Whoops! Somehow that made it into a different branch >.>

Fixed now.